### DISCUSSION TITLE
Support BlueSky Social Media posts

### DISCUSSION THREAD

**Discussion by t0mtaylor** [AzuraCast/AzuraCast#7596] [Ideas]
Support BlueSky Social Media auto posts with now playing information

https://docs.bsky.app/docs/advanced-guides/posts

**Comment by BusterNeece** [comment]
@t0mtaylor moving to a GH issue on #7597, I don't normally do this for enhancements but it's an immediate priority for me to get this out and tested as Bluesky is absolutely taking off (and we're even on it). Follow progress there!
